doc/lua_api.txt

```diff
@@ -3115,7 +3115,15 @@ For the following functions, `v`, `v1`, `v2` are vectors,
 * `vector.cross(v1, v2)`:
     * Returns the cross product of `v1` and `v2`.
 * `vector.offset(v, x, y, z)`:
-    * Returns the sum of the vectors `v` and `{x = x, y = y, z = z}`.
+    * Returns the sum of the vectors `v` and `{x = x, y = y, z = z}`.#
+* `vector.to_string(v)`:
+    * Returns a string `"("..v.x..", "..v.y..", "..v.z..")"`.
```

Comment:
Actually, concatenation and `string.format` can have different results.  I think it would be better to say `Returns a string of the form "(x, y, z)".`, which should be clear enough and is easier to visualize than the concatenation.

Reply:
:+1: Done.